spec_parser/utils.py

```diff
@@ -706,17 +707,20 @@ def _gen_md(self, args: dict) -> None:
     def _gen_rdf(self, g: rdflib.Graph):
 
         cur = URIRef(self.metadata["id"])
-        g.add((cur, RDF.type, OWL["Class"]))
 
         g.add((cur, RDFS.comment, Literal(self.description)))
         g.add((cur, NS0.term_status, Literal(self.metadata.get("Status"))))
 
         # add entries
+        entries = []
```

Comment:
I'd use a `set` here, just to make sure there are no duplicates, but I think the parser already checks that.

Reply:
done